tests/AheadOfTime/check.ps1

```diff
@@ -1,4 +1,10 @@
 Write-Host "AheadOfTime: check1.ps1"
 
+$savedNUGET_PACKAGES=$env:NUGET_PACKAGES
+
+$env:NUGET_PACKAGES=Join-Path $PSScriptRoot "../../artifacts/nuget/AOT/"
```

Comment:
Why is this needed, could you add a comment please?

Reply:
I can do that yes.

Reply:
done